### PR TITLE
Added hover css effects to the footer buttons

### PR DESCRIPTION
**Pull request by vimalprogrammer**
The footer with click-able buttons without hover effects doesn't give the feeling of press, so I added hover opacity effects to the footer buttons.

### CODE REVIEW

**Review by glaukiol1 — APPROVED** [review]
This looks cool! Now we just need to wait on someone with write access to approve this. Thanks for contributing!

**Comment by vimalprogrammer** [comment]
Thank you so much, sir! Eagerly waiting for that precious moment...

**Pull request merged**
